### PR TITLE
API project setup

### PR DESCRIPTION
**Pull request by seerocode**
## Description

- Added base flask and modules setup
- Added virtual env and requirements file

### CODE REVIEW

File: api/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+click==7.1.2
+Flask==1.1.2
+Flask-Cors==3.0.10
+Flask-SQLAlchemy==2.4.4
+gunicorn==20.0.4
+install==1.3.4
+itsdangerous==1.1.0
+Jinja2==2.11.2
+MarkupSafe==1.1.1
+nginx==0.0.1
```
Comment (jnu) [inline api/requirements.txt]: huh! didn't know there was an nginx python package. we're actually going to use nginx as a standalone binary inside the dockerfile. (there'll be a line in the docker file like `apt-get install nginx`). you can remove the nginx dependency here.

thanks for adding gunicorn! we'll set that up at the same time we set up nginx in the dockerfile

**Review by jnu — APPROVED** [review]
first pull request looks great 🎉  woo!!

just remove the nginx dep before merging

File: api/requirements.txt (hunk 1 above)
Comment (seerocode) [reply api/requirements.txt]: Removed Nginx!

**Comment by seerocode** [comment]
Rebased main branch onto PR branch to pull in previous changes

**Pull request merged**